website/docs/docs/cloud/connect-data-platform/connect-onehouse.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connect Onehouse"
+id: connect-onehouse
+sidebar_label: "Connect Onehouse"
+description: "Setup instructions for connecting Onehouse to dbt Cloud"
+pagination_next: null
+---
+
+
+<Constant name="cloud" /> supports connecting to [Onehouse SQL](https://www.onehouse.ai/) using the Apache Spark Connector with the Thrift method.
+
+**Connect to a Onehouse SQL Cluster with the `dbt-spark` adapter.**
```

Comment:
```suggestion
:::note
Connect to a Onehouse SQL Cluster with the [dbt-spark ](/connect-data-platform/connect-apache-spark)adapter.**
:::
```

Seeing the preview - this will look much better